resources/idlharness.js

```diff
@@ -178,26 +178,11 @@ self.IdlArray = function()
      * on a later one.  Save these up and handle them right before we run
      * tests.
      *
-     * .partials is simply an array of objects from WebIDLParser.js'
-     * "partialinterface" production.  .includes maps strings to arrays of
-     * strings, such that
-     *
-     *   A includes B;
-     *   A includes C;
-     *   D includes E;
-     *
-     * results in this["includes"] = { A: ["B", "C"], D: ["E"] }.
-     *
-     * Similarly,
-     *
-     *   interface A : B {};
-     *   interface B : C {};
-     *
-     * results in this["inheritance"] = { A: "B", B: "C" }
+     * Both this.partials and this.includes will be the objects as parsed by
```

Comment:
As hinted at in the commit message, it would be natural to add `this.mixins` and not wrap mixins in `IdlInterface` at all. But not yet.